doc/changes.rst

```diff
@@ -3,6 +3,81 @@
 Changelog
 =========
 
+Version 1.4.0
+-------------
+
+*Released on: 2020/04/06*
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.3739449.svg
+   :target: https://doi.org/10.5281/zenodo.3739449
```

Comment:
I tested this link and the page returned `Error: DOI Not Found`. I am guessing it takes a little while for `doi.org` to populate with the latest release?

Reply:
Ah no, that will only go live once we publish the release. We need the tagged version before we can actually publish on Zenodo so we have to reserve the DOI, release, and then upload the code archive.

Reply:
Ah, that makes sense. Good to know!